docs/go/quickstart/gorillamux-sql.md

```diff
@@ -0,0 +1,113 @@
+---
+id: gorillamux-sql
+title: Sample Application with Gorilla/Mux-SQL
+sidebar_label: Gorilla/Mux-SQL Sample
+tags:
+  - hello-world
+  - go
+  - sdk
+  - tutorial
+---
+
+# URL Shortener Sample App
+
+A sample contact keeper app to test Keploy integration capabilities using [Gorilla Mux](https://pkg.go.dev/github.com/gorilla/mux) and [sql](https://dev.mysql.com/).
+
+## Installation
+
+Navigate to [Installation guide](../../server/server-installation.md) to quickly install and run the keploy server.
+
+### Setup Gorilla/Mux-SQL App
+
+```bash
+git clone https://github.com/keploy/samples-go && cd mux-sql
+go mod download
+```
+
+### Start Keploy Record Mode and start go server
+
+```bash
+export KEPLOY_MODE=record
+```
+
+### Start the postgres SQL server
+
+```bash
+docker-compose up -d
+```
+
+### Run the application
+
+```shell
+go run .
+```
+
+## Generate testcases
```

Comment:
Please change the api calls as per [this](https://github.com/keploy/samples-go/tree/main/mux-sql). You can try these once by running the sample app and making the api calls with the appropriate end points.